auth/src/test/java/com/firebase/ui/auth/ui/email/RecoverPasswordActivityTest.java

```diff
@@ -1,69 +0,0 @@
-/*
- * Copyright 2016 Google Inc. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License. You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software distributed under the
- * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
- * express or implied. See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.firebase.ui.auth.ui.email;
-
-import android.content.Intent;
-import android.widget.Button;
-
-import com.firebase.ui.auth.BuildConfig;
-import com.firebase.ui.auth.R;
-import com.firebase.ui.auth.testhelpers.AuthHelperShadow;
-import com.firebase.ui.auth.testhelpers.AutoCompleteTask;
-import com.firebase.ui.auth.testhelpers.TestConstants;
-import com.firebase.ui.auth.testhelpers.TestHelper;
-
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.robolectric.Robolectric;
-import org.robolectric.RobolectricTestRunner;
-import org.robolectric.RuntimeEnvironment;
-import org.robolectric.annotation.Config;
-
-import java.util.Collections;
-
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
-
-@RunWith(RobolectricTestRunner.class)
-@Config(constants = BuildConfig.class, sdk = 25)
-public class RecoverPasswordActivityTest {
```

Comment:
Tests! I don't think this one is particularly useful, but I haven't written any new ones yet. I was think of writing tests for:
- `SingleLiveEvent`
- `AutoClearSingleLiveEvent`
- `ViewModelBase` for completness
- `RecoverPasswordHandler`?

Any thoughts?

Reply:
Those would be a great start, especially the `LiveEvent` classes.  I am fine if you want to make that a separate PR though.

Reply:
Yeah, I think a separate PR would be good so we can really focus on writing quality tests and setting a standard for the rest of the codebase.